cli/ceph/rgw/realm.py

```diff
@@ -10,13 +10,18 @@ def __init__(self, nodes, base_cmd):
 
         self.base_cmd = f"{base_cmd} realm"
 
-    def bootstrap(self, **kw):
+    def bootstrap(self, input=None, **kw):
         """
         Performs rgw realm bootstrap
         Args:
             kw: Key/value pairs of configuration information to be used in the test.
         """
-        cmd = f"{self.base_cmd} bootstrap {build_cmd_from_args(**kw)}"
+        cmd = f"{self.base_cmd} bootstrap "
```

Comment:
@vinpapnoi Why do we need to remove build_cmd_from_args(**kw)?

Reply:
It is moved below. Needed to include "input" variable in the command.

Reply:
I think you do not have to add extra argument to this method, you can still use build_cmd_from_args, and pass  --in-file as mentioned in ceph rgw realm bootstrap --help:
 -i INPUT_FILE, --in-file INPUT_FILE
 
 @vamahaja correct me if I'm wrong.

Reply:
Thanks for the suggestion @adityaramteke 
I tried running the test with your suggested changes and it worked.